testgrid/config/config.yaml

```diff
@@ -223,6 +223,8 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-sig-cli
 - name: ci-kubernetes-e2e-gci-gce-ingress
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-ingress
+  alert_stale_results_hours: 24
+  num_failures_to_alert: 3
```

Comment:
Seems like this is the same as default, do we need to redefine it?
https://github.com/kubernetes/test-infra/blob/e75dd6e6fed04c554b5f254f4198f944bf6ded49/testgrid/config/config.yaml#L24

Reply:
sry didn't realize there were defaults, I will update